rallyrolebot/cogs/update_cog.py

```diff
@@ -170,3 +171,44 @@ async def update(self):
                 + str(member_count)
                 + " members."
             )
+
+    @commands.command(help="updates your wallet balance / roles immediately")
+    @commands.guild_only()
+    async def set_wallet_id(self,ctx:commands.Context):
+        if not isinstance(ctx.author,discord.Member):
+            raise errors.MemberNotFound("command executer is not member")
```

Comment:
Since you've used the `@commands.guild_only()` decorator, `ctx.author` is guaranteed to be a member from a guild. (No need to check if is `discord.Member`)